docs/hub/local-apps.md

```diff
@@ -0,0 +1,95 @@
+# Use AI Models Locally
+
+You can run AI models from the Hub locally on your machine. This means that you can benefit from these advantages:
+
+- **Privacy**: You won't be sending your data to a remote server.
+- **Speed**: Your hardware is the limiting factor, not the server or connection speed.
+- **Control**: You can configure models to your liking.
+- **Cost**: You can run models locally without paying for an API provider.
+
+## How to Use Local Apps
+
+Local apps are applications that can run Hugging Face models directly on your machine. To get started:
+
+1. **Enable local apps** in your [Local Apps settings](https://huggingface.co/settings/local-apps).
+
+![Local Apps](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/local-apps/settings.png)
+
+1. **Choose a supported model** from the Hub by searching for it. You can filter by `app` in the `Other` section of the navigation bar: 
+
+![Local Apps](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/local-apps/search_llamacpp.png)
+
+3. **Select the local app** from the "Use this model" dropdown on the model page.
+
+![Local Apps](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/local-apps/button.png)
+
+4. **Copy and run** the provided command in your terminal.
```

Comment:
technically in many (most?) cases the link is a deep-link to open the app with the specified model, not a command to copy/paste

(cc @gary149 too for viz)